### PR TITLE
[DOCFIX] Update Chinese version of Contributor-Getting-Started.md

### PR DESCRIPTION
**Pull request by thanosaw**
Updates the old outdated Contribution guide to match the English version.

### CODE REVIEW

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 1]
```diff
@@ -1,227 +1,262 @@
...
+## 软件要求
 
-```console
-$ java -version
-```
+最基本的需要是一台安装了Mac OS X或Linux的电脑，目前版本的Alluxio尚未对Windows系统提供支持。
 
-如果你还未安装Java，从[Java SDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)下载并安装。
+我们建议您先按照基于源码编译Alluxio
+[指南克隆并编译Alluxio源代码。]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}).
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:19]:
```suggestion
[指南克隆并编译Alluxio源代码]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }})。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 2]
```diff
@@ -1,227 +1,262 @@
...
+### 帐户准备
 
-Alluxio使用Git分布式版本控制系统来管理源码，因此需要安装Git。
+#### Github帐户
 
-如果你还未安装`git`，请先进行[安装git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)。
+您需要有一个GitHub帐户才能向Alluxio代码仓库贡献代码。
 
-### GitHub账户
+您需要知道您的GitHub电子邮件地址才能进行贡献。 您可以在个人资料电子邮件设置中找到你的邮件地址。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:34]:
```suggestion
您需要知道您的GitHub电子邮件地址才能进行贡献。 您可以在[个人资料电子邮件设置](https://github.com/settings/emails)中找到你的邮件地址。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 3]
```diff
@@ -1,227 +1,262 @@
...
 
-你需要一个GitHub账户来贡献源码，如果还没有，请先[注册一个GitHub账户](https://github.com/)。
+在向Alluxio提交代码之前，您应该确认你的Git电子邮件设置正确。
+[请访问有关设置电子邮件的说明。](https://help.github.com/articles/setting-your-email-in-git/).
 
-另外你还需要知道[你的GitHub账户绑定了哪个邮箱](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)。
+## 分叉Alluxio代码仓库
 
-## Fork Alluxio源代码库
+为了向Alluxio贡献代码，您先需要叉Alluxio代码仓库（repo）。 如果您还没分叉代码仓库，您可以访问[Alluxio代码仓库](https://github.com/Alluxio/alluxio)并按页面右上角的Fork按钮。 之后，您将拥有自己的Alluxio代码仓库分支。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:43]:
```suggestion
为了向Alluxio贡献代码，您先需要分叉Alluxio代码仓库。 如果您还没分叉代码仓库，您可以访问[Alluxio代码仓库](https://github.com/Alluxio/alluxio)并按页面右上角的Fork按钮。 之后，您将拥有自己的Alluxio代码仓库分支。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 4]
```diff
@@ -1,227 +1,262 @@
...
-在fork Alluxio源代码库后，你需要从该fork创建一个本地的副本，这会将该fork里的文件拷贝到你的本地电脑。使用以下命令创建副本：
+分叉Alluxio代码仓库之后，应创建分叉代码仓库的本地克隆。 这会将您分叉的文件复制到您的计算机上。 您可以使用以下命令克隆您的分叉：
 
 ```console
 $ git clone https://github.com/YOUR-USERNAME/alluxio.git
 $ cd alluxio
 ```
 
-这会将副本拷贝在`alluxio/`目录下。
+这将在alluxio/目录下创建克隆。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:53]:
```suggestion
这将在`alluxio/`目录下创建克隆。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 5]
```diff
@@ -1,227 +1,262 @@
...
 
-为了将远程的Alluxio源码改动更新到你本地的副本中，你需要创建一个指向远程Alluxio源代码库的源。在刚拷贝的副本的目录下，运行：
+要将更改从开源Alluxio原始代码库提取到克隆中，您需要创建一个指向Alluxio原始代码库的新远程代码存储库。 要添加新的远程代码存储库，请在新创建的克隆目录中运行：
 
 ```console
 $ git remote add upstream https://github.com/Alluxio/alluxio.git
 ```
 
-运行以下命令可以查看远程仓库的url：
+这将创建一个称为上游的远程代码仓库指向Alluxio原始代码仓库。 您可以用以下命令查看远程代码仓库的URL:
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:61]:
```suggestion
这将创建一个称为`upstream`的远程代码仓库指向Alluxio原始代码仓库。 您可以用以下命令查看远程代码仓库的URL: 
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 6]
```diff
@@ -1,227 +1,262 @@
...
 ```console
 $ git remote -v
 ```
 
-这会显示`origin`（你的fork）以及`upstream`（Alluxio仓库）的url。
-
-## 配置Git邮箱
-
-在向Alluxio提交commit之前，你需要先确认你的Git邮箱设置正确，请参考该[邮箱设置指南](https://help.github.com/articles/setting-your-email-in-git/)。
+这会显示远程代码仓库的URL，包括您的分叉和上游代码仓库（Alluxio原始代码仓库）。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:67]:
```suggestion
这会显示远程代码仓库的URL，包括`origin`（您的分叉）和`upstream`（Alluxio原始代码仓库）。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 7]
```diff
@@ -1,227 +1,262 @@
...
 
 ```console
-$ mvn clean install
+$ man clean install
 ```
 
-该命令会编译整个Alluxio，并且运行所有测试，这可能会花费几分钟。
+这将编译所有的Alluxio，并运行所有测试。取决于您的硬件，这可能需要几分钟到半小时才能完成。
+
+如果您想重新编译而不运行所有检查和测试，您可以运行：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:82]:
```suggestion
如果您想重新编译而不运行所有检查和测试，运行：
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 8]
```diff
@@ -1,227 +1,262 @@
...
+$ mvn -T 2C clean install -DskipTests -Dmaven.javadoc.skip -Dfindbugs.skip \
+  -Dcheckstyle.skip -Dlicense.skip
 ```
 
-这应该用不到一分钟。
+此编译所花时间应少于1分钟。
 
-这里有更多[编译Alluxio源代码]({{ '/cn/contributor/Building-Alluxio-From-Source.html' | relativize_url }})。
+这里有更多
+[细节编译Alluxio源代码。]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}).
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:93]:
```suggestion
[编译Alluxio源代码的细节]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }})。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 9]
```diff
@@ -1,227 +1,262 @@
...
 
-## 领取一个New Contributor任务
+## 领取一个新贡献者任务
 
-Alluxio的ticket中有许多不同的等级，它们分别是：**New Contributor**、**Beginner**、**Intermediate**、 **Advanced**。新的开发者应该在进行更高级的任务之前先完成两个**New Contributor**任务。**New Contributor**任务非常容易，不需要了解过多的关于代码的细节；**Beginner**任务通常只需要修改一个文件；**Intermediate**任务通常需要修改多个文件，但都在同一个包下；**Advanced**任务通常需要修改多个包下的多个文件。
+Alluxio事件表具有多个级别。级别为：**新贡献者**，**初学者**，**中级**，**高级**。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:97]: The previous version sounds better. And we don't need to translate **New Contributor**、**Beginner**、**Intermediate**、 **Advanced** here

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 10]
```diff
@@ -1,227 +1,262 @@
...
 
-请浏览任何的未关闭的[New Contributor Alluxio Tasks](https://github.com/Alluxio/new-contributor-tasks/issues)并找到一个还未被他人领取的任务，你可以点击**Assign to yourself**链接来领取该ticket。另外你应该在你开始进行该任务之前先领取它，从而其他人知道你在完成这个ticket。
+看一些在“打开”状态的[“新的Contributor Alluxio任务”](https://github.com/Alluxio/new-contributor-tasks/issues)，然后找一个没分配的任务。为了自我分配或接收调查表，请在问题中留下评论，例如/ assign @yourUserName，以表明您正在处理该问题。在开始解决问题单之前，您应该将问题单分配给自己，以便社区中的其他人知道您正在处理和完成问题单。
 
-注意所有Github Issue都有一个对应的数字。当你创建一个pull request的时候，
-请在该Issue的描述中加入
-"Fixes Alluxio/new-contributor-tasks#123"，"Fixed Alluxio/new-contributor-tasks#123"，
-"Fix Alluxio/new-contributor-tasks#123"，"Closes Alluxio/new-contributor-tasks#123"，
-"Closed Alluxio/new-contributor-tasks#123" 或者 "Close Alluxio/new-contributor-tasks#123"。
+Github上所有新的贡献者级别的问题均分配有一个编号。可以在问题标题后找到该数字，例如"问题＃123"。创建拉取请求以解决问题时，应将链接/指针添加回问题本身。为此，必须在拉取请求描述中添加特定描述。例如，如果您的问题编号为＃123，则应在拉取请求说明中包括以下内容之一：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:105]: previous version is better

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 11]
```diff
@@ -1,227 +1,262 @@
...
+  * `修复Alluxio / new-contributor-tasks＃123`
+  * `修复了Alluxio / new-contributor-tasks＃123`
+  * `修复Alluxio / new-contributor-tasks＃123`
+  * `关闭Alluxio /新贡献者任务＃123`
+  * `已关闭Alluxio / new-contributor-tasks＃123`
+  * `关闭Alluxio / new-contributor-tasks＃123`
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:111]: Keep them in english
```suggestion
  * `Fixes Alluxio/new-contributor-tasks#123`
  * `Fixed Alluxio/new-contributor-tasks#123`
  * `Fix Alluxio/new-contributor-tasks#123`
  * `Closes Alluxio/new-contributor-tasks#123`
  * `Closed Alluxio/new-contributor-tasks#123`
  * `Close Alluxio/new-contributor-tasks#123`
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 12]
```diff
@@ -1,227 +1,262 @@
...
+找到一个任务后，回到终端并转到本地克隆的目录下。 
+您可以开始解决问题了！
 
-在领取ticket之后，切换到终端下，进入本地副本的目录，现在，我们开始动手修复吧！
+为了提交更改给Alluxio，最佳实践是在一个独有分支中完成对针对一个问题您的所有更改。因此，以下内容将向您介绍如何创建分支。
 
-要向Alluxio提交一个修改，最好是为每一个ticket单独建一个分支，并且在该分支下进行修改。以下命令将展示如何创建一个新的分支。
-
-首先，确保你在本地副本的`master`分支下，运行以下命令切换到`master`分支：
+首先，确保您在您克隆的master分支中。运行以下命令切换到master分支：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:120]: keep the quotes

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 13]
```diff
@@ -1,227 +1,262 @@
...
 
 ```console
 $ git checkout master
 ```
 
-接着，你应当确保你的`master`分支里的代码与最新的Alluxio源码同步，可以通过以下命令获取所有的代码更新：
+然后，您应确保您的master分支与不断发展的Alluxio原始代码库中的最新更改保持同步。运行以下命令来提取项目中所有最新更改：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:126]: quotes

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 14]
```diff
@@ -1,227 +1,262 @@
...
 
 ```console
-git pull upstream master
+$ git pull upstream master
 ```
 
-这将会获取到Alluxio项目中所有的更新，并合并到你的本地`master`分支里。
+这会将所有更改从上游代码仓库的master分支拉到本地master分支。在此示例中，上游代码仓库就是Alluxio开源项目。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:132]:
```suggestion
这将会从`upstream`仓库的`master`分支中获取到所有的更新，并合并到你的本地`master`分支里。
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 15]
```diff
@@ -1,227 +1,262 @@
...
 ```
 
-这会创建该分支，并且切换到该分支下。现在，你可以修改相应的代码来完成该任务啦！
+这将创建分支，并切换到该分支。您可以修改必要的代码以解决该问题。
 
-## 提交本地commit
+### 创建本地提交
 
-在你处理该任务时，可以为修改的代码提交本地的commit，这在你完成了一个阶段性的修改时特别有用。运行以下命令将一个文件标记为准备提交阶段：
+A在解决问题单时，您可以创建代码的本地提交。 当您完成一部分很明确的阶段性修改后，这将很有用。 您可以使用以下命令将文件添加到准备提交阶段：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:145]: Previous version is better, can you consolidate?

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 16]
```diff
@@ -1,227 +1,262 @@
...
 
 ```console
 $ git add <file to stage>
 ```
 
-一旦所有需要的文件都进入准备提交阶段后，可以运行以下命令提交包含这些修改的一个commit：
+在所有合适的文件都准备好提交之后，您可以使用以下方法，创建这些可修改的本地提交：
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:151]:
```suggestion
在所有合适的文件都准备好提交之后，可以运行以下命令提交包含这些修改的一个commit：
```

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 17]
```diff
@@ -1,227 +1,262 @@
...
 
-## 提交一个Pull Request
+ 
+如果您想要更多有关创建提交的信息，请访问有关如何创建提交的[说明](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository)。
 
-在你完成了该Issue的所有修改后，就马上要向Alluxio项目提交一个pull request啦！这里是一个[提交一个pull request的详细指南](https://help.github.com/articles/using-pull-requests/)，不过以下是通常的做法。
+### 提交一个拉取请求
 
-在你提交了所有需要的本地commit后，你可以将这些commit推送到你的GitHub源代码库中。对于**awesome_feature**分支，运行以下命令推送到GitHub上：
+完成所有解决问题所需修改后，您可以向Alluxio项目提交拉取请求了！这是[有关提交拉取请求的详细说明](https://help.github.com/articles/using-pull-requests/)，但是以下是通常做法。
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:164]: 拉取请求 -> Pull request, through this page

File: docs/cn/contributor/Contributor-Getting-Started.md [hunk 18]
```diff
@@ -1,227 +1,262 @@
...
 ```
 
-这将会把你本地的**awesome_feature**分支下的所有commit推送到你的GitHub上Alluxio fork的**awesome_feature**分支中。
+这会将您在本地分支**awesome_feature**中的所有新提交，推送到您的Alluxio分支中的GitHub中的**awesome_feature**分支中。
+
+将所有更改推送到分支后，请访问您的Alluxio的GitHub分支。通常，Github会显示您的哪个分支最近已更新，但如果未更新，直接选择要提交拉取请求的分支（在本示例中为**awesome_feature**），然后按New Pull Request按钮。
+
+在提交新拉取请求页面中，基础分支应为Alluxio / alluxio，基础分支应为master。头分支将是您的分支，而compare分支应是您要提交拉取请求的分支（在此示例中为awesome_feature）。
+
+#### 拉取请求标题
```
Comment (apc999) [inline docs/cn/contributor/Contributor-Getting-Started.md:178]: Pull request

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Comment by ZacBlanco** [comment]
@apc999 this PR introduces a failure in the documentation check due to referencing of a broken link. It needs to be fixed or the build will fail consistently for any new doc PRs.

**Pull request merged**